include/aikido/trajectory/util.hpp

```diff
@@ -45,11 +45,13 @@ UniqueInterpolatedPtr concatenate(
 ///
 /// \param[in] traj Input trajectory
 /// \param[in] referenceState Reference state
+/// \param[out] distance Distance to the closest state
```

Comment:
Should this outparam be before the timeStep inparam?

Reply:
Yes, since the timeStep has a default value.